_config.yml

```diff
@@ -6,11 +6,11 @@ title: AutoML Benchmark
 
 # Where the page will be deployed.
 # If it's https://foo.com/bar, just write "https://foo.com"
-url: "https://jonathankrauss.github.io/AutoML-Benchmark/"
+url: "https://hannomaximilian.github.io/AutoML-Benchmark/"
```

Comment:
Please ignore this change or return it to the original value, so it keeps pointing to the correct webpage.